Corona-Warn-App/src/main/java/de/rki/coronawarnapp/deadman/DeadmanNotificationOneTimeWorker.kt

```diff
@@ -0,0 +1,49 @@
+package de.rki.coronawarnapp.deadman
+
+import android.content.Context
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import com.squareup.inject.assisted.Assisted
+import com.squareup.inject.assisted.AssistedInject
+import de.rki.coronawarnapp.util.worker.InjectedWorkerFactory
+import de.rki.coronawarnapp.worker.BackgroundConstants
+import timber.log.Timber
+
+/**
+ * One time background deadman notification worker
+ *
+ * @see DeadmanNotificationScheduler
+ */
+class DeadmanNotificationOneTimeWorker @AssistedInject constructor(
+    @Assisted val context: Context,
+    @Assisted workerParams: WorkerParameters,
+    private val sender: DeadmanNotificationSender
+) :
+    CoroutineWorker(context, workerParams) {
+
+    /**
+     * Work execution
```

Comment:
Comments are good, but should help the developer better understand the code, e.g. explain why certain decisions were made. Comments that say the same as a function name just add noise to the code when reading it.

I'd say this is a superfluous comment, what do you think?

Reply:
Agree, removed superfluous comments.